### PR TITLE
Feat: New TopPage component

### PR DESCRIPTION
**Pull request by arielsrodriguez**
## Summary

New TopPage component

## Task

- https://dd360.atlassian.net/jira/software/projects/PD/boards/82?assignee=6397d027fbf54baf29036449&selectedIssue=PD-1169

## Affected sections

- src/components/TopPage/TopPage.test.tsx
- src/components/TopPage/TopPage.tsx
- src/stories/TopPage.stories.tsx

## How did you test this change?

- Manually tested with Storybook 🎨
- Added test to TopPage component 🤖
- All tests passed ✅

### CODE REVIEW

**Comment by tbor00** [comment]
Why this PR is in draft?

File: src/components/TopPage/TopPage.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
+import React, { useCallback } from 'react'
+import { composeClasses } from 'lib/classes'
+import Breadcrumbs, { BreadcrumbsProps } from 'components/Breadcrumbs'
+import Flex from 'components/Layout/Flex'
+import Circle from 'components/Circle'
+import { Button } from 'components/Buttons'
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:6]:
```suggestion
import  Button  from 'components/Buttons'
```

File: src/components/TopPage/TopPage.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
+import React, { useCallback } from 'react'
+import { composeClasses } from 'lib/classes'
+import Breadcrumbs, { BreadcrumbsProps } from 'components/Breadcrumbs'
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:3]:
```suggestion
import Breadcrumbs, type { BreadcrumbsProps } from 'components/Breadcrumbs'
```

File: src/components/TopPage/TopPage.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  onClick: () => void
+  icon: React.ReactNode
+  variant: 'primary' | 'secondary' | 'tertiary'
+  label: string
+  isDisabled?: boolean
+}
+
+interface ITab {
+  value: number
+  setValue: (value: number) => void
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:21]: maybe can be setStateAction?

File: src/components/TopPage/TopPage.tsx [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+  optionsBreadcrumbs?: BreadcrumbsProps['options']
+  lastUpdate?: {
+    translation: 'es' | 'en'
+    date: Date
+  }
+  title: string
+  description?: string
+  callToActionsButtons?: IActionButton[]
+  children: React.ReactNode
+  callToActionIcon?: {
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:38]: Move to a diff type

File: src/components/TopPage/TopPage.tsx [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+interface ITab {
+  value: number
+  setValue: (value: number) => void
+  items: {
+    label: string
+    disabled?: boolean
+  }[]
+}
+
+interface ITopPage {
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:28]:
```suggestion
interface TopPageProps {
```

File: src/components/TopPage/TopPage.tsx [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+            <Divider className="mt-3" light />
+          )}
+        </div>
+      </div>
+      <div className="my-6">{children}</div>
+    </div>
+  )
+}
+
+export default TopPage
```
Comment (tbor00) [inline src/components/TopPage/TopPage.tsx:210]: And don't miss the `index.ts` to export all

**Comment by arielsrodriguez** [comment]
> Why this PR is in draft?

I’ve removed it from draft

File: src/components/TopPage/TopPage.tsx (hunk 1 above)
Comment (arielsrodriguez) [reply src/components/TopPage/TopPage.tsx:6]: I have this error when making the change:
Module '"/Users/ariel/bui/src/components/Buttons/index"' has no default export. Did you mean to use 'import { Button } from "/Users/ariel/bui/src/components/Buttons/index"' instead?haz

**Review by genixxavier — APPROVED** [review]

File: src/components/TopPage/TopPage.tsx [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+const translationLastUpdate = (translation: 'es' | 'en') => {
+  if (translation === 'es') return 'Última actualización:'
+
+  return 'Last update:'
+}
+
+const formatDate = (date: Date, language: 'en' | 'es') => {
+  const day = date?.getDate?.()?.toString()?.padStart(2, '0')
+
+  const monthNames = {
```
Comment (Misael840) [inline src/components/TopPage/TopPage.tsx:57]: Maybe we can move this const to a utils file to reuse

**Review by Misael840 — APPROVED** [review]

**Pull request merged**